### ISSUE TITLE
Invent a better name

### ISSUE THREAD

**Issue by PeterGy** [EinarElen/prototype-geometry#11]

**Comment by PeterGy** [comment]
We could use support_plate or bar_support_plate or bar_mount. Or maybe you have a better idea?

**Comment by EinarElen** [comment]
If this is the plate that the scintillators are attached to, the name should make that relationship clear. Otherwise, I think I prefer the mount (but perhaps mounting plate?)

**Comment by PeterGy** [comment]
Yes, I mean those plates. I'll go with bar_mounting_plate for now unless/until we can come up with someting better.
